utils/programUtils.js

```diff
@@ -28,6 +28,32 @@ const generateProgram = () => {
   };
 };
 
+const generateProgram2 = () => {
```

Comment:
what is the difference between this and the existing `generateProgram2`?

Reply:
This is generating a different program that will be used to edit for the edit program test. It ended up in this pull request accidentally.

Reply:
ok :) not a big deal that this is here in this PR, just didn't see where it was used here :P